src/shadowbox/server/server_access_key.ts

```diff
@@ -70,14 +67,15 @@ function makeAccessKey(hostname: string, accessKeyJson: AccessKeyJson): AccessKe
       accessKeyJson.id, accessKeyJson.name, accessKeyJson.metricsId, proxyParams);
 }
 
-function makeAccessKeyJson(accessKey: AccessKey): AccessKeyJson {
+function accessKeySerializedJson(accessKey: AccessKey): AccessKeyJson {
```

Comment:
In your previous comment you noted taht both of these functions serialize, which is a good point.  I think accessKeyToPersistedJson satisfies both of our concerns.